mdsa/pcoa.py

```diff
@@ -172,18 +173,22 @@ def pcoa(distance_matrix, algorithm, num_dimensions_out=10):
         # Only select k (num_dimensions_out) first eigenvectors
         # and their corresponding eigenvalues from the sorted array
         # of eigenvectors / eigenvalues
-        if len(eigenvectors) > num_dimensions_out:
-            eigenvectors = eigenvectors[:num_dimensions_out]
+
+        if len(eigenvalues) > num_dimensions_out:
```

Comment:
This is the same commit that's in the previous fix PR #25 

```python
eigenvectors = [
    [0,0,0],
    [2,2,2], 
    [1,1,1]
]
num_dimensions_out = 3
```

Before:
```
# wrong, as far as I understand
eigenvectors = eigenvectors[:num_dimensions_out]

# output:
#[
#    [0,0,0],
#    [2,2,2]
#]
```

After fix:
```
# correct, as far as I understand
eigenvectors = eigenvectors[:, :num_dimensions_out]

# output:
#[
#    [0,0],
#    [2,2],
#    [1,1]
#]
```